src/index.d.ts

```diff
@@ -19,5 +19,5 @@ declare namespace uvu {
 type Context = Record<string, any>;
 export const test: uvu.Test<Context>;
 export type Callback<T=Context> = uvu.Callback<T>;
-export function suite<T=Context>(title?: string, suite?: T): uvu.Test<T>;
+export function suite<T=Context>(title?: string, context?: Partial<T>): uvu.Test<T>;
```

Comment:
What do you think about this `Partial` @lukeed? I wanted this to be stuff like sync config, that get's enhanced by future before's... But don't want to have TS say `context?.myThing` always, but making the property be `myThing?: string` for example.

Reply:
It can't be enforced here. It's just forwarding whatever _you_ decide the `context` contains. Entirely up to you & uvu types shouldn't mess with that.